src/octoprint/util/__init__.py

```diff
@@ -1078,27 +1078,11 @@ def __exit__(self):
             self.cleanup()
 
 
-def bom_aware_open(filename, encoding="ascii", mode="r", **kwargs):
-    import codecs
-
+def bom_aware_open(filename, encoding="utf-8", mode="r", **kwargs):
```

Comment:
Given that the long time goal here should be to get rid of `bom_aware_open` altogether, how about:

```suggestion
@deprecated("Please use io.open with '-sig' encoding instead", since="1.8.0")
def bom_aware_open(filename, encoding="utf-8-sig", mode="r", **kwargs):
```